content/en/blog/_posts/2023-10-31-Gateway-API-GA/index.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: blog
+title: "Gateway API v1.0: GA Release"
```

Comment:
Maybe "Gateway API v1.0: GA Is in the Name" would be a more engaging title?

Reply:
Fine with me

Reply:
This came out of a discussion with @youngnick yesterday, building on the alliteration idea: ***G***ateway ***A***PI == GA (first letters of each word in the name). With that said, I'm definitely not set on this title and agree that it could be confusing.

Reply:
I think given the timeframe, just pick your favorite I'm easy about this.

Reply:
Although boring, what we already have is accurate and easy enough to understand, so I'm fine with just sticking with that. Also don't have a strong opinion though.

Reply:
🤦  whoops, in any case, I'm good with what we've got here.